bindings/swift/native/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[workspace]
+
+[package]
+name = "iota-wallet-swift-binding"
+version = "0.1.0"
+authors = ["Thoralf Müller <thoralf.mueller@iota.org", "Pasquale Ambrosini <pasquale.ambrosini@gmail.com"]
+edition = "2018"
+
+[lib]
+name = "wallet"
```

Comment:
Unless there is necessity I don't think this is needed (if we use `iota-wallet` as the name of this (bindings) crate).